00-introduction.md

```diff
@@ -0,0 +1,133 @@
+(which turns out to be pretty complicated once you start looking beyond
+the standardized Western classroom).
+
+Within the broad scope of educational pyschology, 
+this training will take a narrow "cognitivist" perspective on education.
+We can afford to do this because most of our learners are privileged:
+well-fed, physically safe, already well-educated, highly motivated, etc.
+
+> ## A Broader Perspective 
+> 
+> There are many other perspectives outside cognitivist theory ---
+> see [this site](http://www.learning-theories.com/) for summaries ---
+> and it's worth exploring them for insights like this:
+> "If poor inner-city children consistently outscored children
+> from wealthy suburban homes on standardized tests,
+> is anyone naive enough to believe that we would still insist on using these tests
+> as indicators of success?"
+> (Kenneth Wesson, in Littky and Grabelle's *The Big Picture*)
+{: .callout}
+ 
+## Instructional Design
+
+While what we know about educational psychology can constrain teaching,
+it doesn't dictate it.
+In particular,
+there are often several plausible approaches to teaching
+that are consistent with what we know about how brains learn.
+In order to decide between them,
+we also need to explore *instructional design*,
+which is the study of how to create learning materials.
+If educational psychology is the science,
+ID is the engineering:
+it's how our scientific understanding is put into practice on the ground.
+
+For example,
+most children today are taught to read using a bottom-up technique called
+[phonics](http://en.wikipedia.org/wiki/Phonics),
+which introduces them to the sounds of letters,
+then puts those letters and their sounds together to make simple words like "bug" and "run",
+then assembles the words into sentences and stories.
+But another technique called [whole language](http://en.wikipedia.org/wiki/Whole_language)
+takes a different approach.
+Instead of starting with letters,
+its proponents teach children to recognize entire words
+so that they get the positive feedback of being able to read whole stories as early as possible.
+The basic idea is that once children find reading rewarding,
+they'll be motivated to go back and figure out the letters.
+
+Both approaches are consistent with what we know about how children learn,
+so we need to study them in action to determine which is better.
+The findings have been contradictory:
+while some studies find that phonics works best,
+others show that whole language yields better results.
+
+One explanation is that a third factor is at play.
+If children sense that their teachers are enthusiastic about something,
+they will respond to that enthusiasm and learn better.
+If they feel their teachers are just marking time,
+though,
+they will emulate that instead.
+Thus,
+when studies are done by people who are proponents of a new technique
+(as they often are),
+they are likely to produce positive results,
```

Comment:
positive results as a consequence of being enthusiastic about teaching that way,